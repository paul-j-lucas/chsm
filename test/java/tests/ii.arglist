-j | int x, int y | 0
