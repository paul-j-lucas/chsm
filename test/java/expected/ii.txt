decl="int $" name="x"
decl="int $" name="y"
